server/migrations/auth/authenticate.js

```diff
@@ -0,0 +1,83 @@
+import User from '@models/User';
```

Comment:
Where is this file coming from ?

Reply:
I have no idea how it found it's way there. I have removed it now, must have been during the issue I informed you earlier. Thank you